lang/java/reef-applications/reef-vortex/src/main/java/org/apache/reef/vortex/driver/DefaultVortexMaster.java

```diff
@@ -50,14 +51,18 @@
     this.pendingTasklets = pendingTasklets;
   }
 
-  /**
-   * Add a new tasklet to pendingTasklets.
-   */
```

Comment:
I understand why you removed this (because it overrides `VortexMaster.enqueueTasklet()`, and the method already has one, right?). But, still we have comments in other methods in this class. How does it sound to tackle this issue in another PR?